src/element/builders.rs

```diff
@@ -101,9 +82,19 @@ impl SExpBuilder {
         self
     }
 
+    /// Builds a [`Sequence`] with the previously specified elements.
+    pub fn build(self) -> Sequence {
+        self.values.into()
+    }
+
+    /// Builds a [`List`] with the previously specified elements.
+    pub fn build_list(self) -> List {
+        List(self.build())
+    }
+
     /// Builds a [`SExp`] with the previously specified elements.
-    pub fn build(self) -> SExp {
-        SExp::new(self.values)
+    pub fn build_sexp(self) -> SExp {
+        SExp(self.build())
```

Comment:
🗺️ The unified `SequenceBuilder` type now offers `build()`, `build_list()`, and `build_sexp()` methods that construct the expected types.